specifyweb/specify/migration_utils/update_schema_config.py

```
@@ -198,3 +254,1146 @@ def revert_table_field_schema_config(table_name, field_name, apps = apps):
+        Splocalecontainer = apps.get_model('specify', 'Splocalecontainer')
+        Splocalecontaineritem = apps.get_model('specify', 'Splocalecontaineritem')
+
+        for table, fields in MIGRATION_0034_UPDATE_FIELDS.items():
+            containers = Splocalecontainer.objects.filter(name=table.lower())
+            for container in containers:
+                for (field_name, _, _) in fields:
+                    items = Splocalecontaineritem.objects.filter(
+                        container=container,
+                        name=field_name.lower()
+                    )
+                    for item in items:
```

Ruff (0.15.13)

[warning] Loop control variable `item` not used within loop body

Rename unused `item` to `_item`

(B007)